### ISSUE TITLE
don't include same README for packaging

### ISSUE THREAD

**Issue by fgregg** [datamade/django-geomultiplechoice#11]
The README for this repo is very good, but includes much information that is not relevant for folks who will be installing from pypi.

- The example does not work, because you are not also shipping the Makefile (and that would be strange if you were)
- The developing instructions are not really appropriate for this.

**Comment by beamalsky** [comment]
Good point. @jeancochrane and I went back and forth about including the `example` implementation in this repo at all, and I'm glad I did, but I agree that there's no reason to have it in the package. I've implemented those changes in #13.
